binders/kafka-binder/spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderConfigurationTest.java

```diff
@@ -16,42 +16,25 @@
 
 package org.springframework.cloud.stream.binder.kafka;
 
-import java.lang.reflect.Field;
-
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.api.extension.ExtendWith;
 
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.autoconfigure.kafka.KafkaAutoConfiguration;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.cloud.stream.binder.kafka.config.KafkaBinderConfiguration;
-import org.springframework.kafka.support.ProducerListener;
-import org.springframework.test.context.junit.jupiter.SpringExtension;
-import org.springframework.util.ReflectionUtils;
+import org.springframework.test.util.ReflectionTestUtils;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
  * @author Ilayaperumal Gopinathan
+ * @author Chris Bono
  */
-@ExtendWith(SpringExtension.class)
```

Comment:
We need to do a later sweep and remove `@ExtendWith(SpringExtension.class)` from all `@SpringBootTest` as SBT includes that annotation.

Reply:
To be handled in #2371